[2024-05-24T08:37:31.905+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-24T08:37:31.921+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.second_task scheduled__2024-05-14T00:00:00+00:00 [queued]>
[2024-05-24T08:37:31.926+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.second_task scheduled__2024-05-14T00:00:00+00:00 [queued]>
[2024-05-24T08:37:31.927+0000] {taskinstance.py:2306} INFO - Starting attempt 3 of 6
[2024-05-24T08:37:31.935+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): second_task> on 2024-05-14 00:00:00+00:00
[2024-05-24T08:37:31.939+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1481) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-24T08:37:31.941+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'second_task', 'scheduled__2024-05-14T00:00:00+00:00', '--job-id', '53', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpbmim18f0']
[2024-05-24T08:37:31.944+0000] {standard_task_runner.py:63} INFO - Started process 1491 to run task
[2024-05-24T08:37:31.944+0000] {standard_task_runner.py:91} INFO - Job 53: Subtask second_task
[2024-05-24T08:37:31.971+0000] {task_command.py:426} INFO - Running <TaskInstance: our_first_dag_v2.second_task scheduled__2024-05-14T00:00:00+00:00 [running]> on host 6bbd6353fbe6
[2024-05-24T08:37:32.018+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gauti' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2024-05-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-14T00:00:00+00:00'
[2024-05-24T08:37:32.021+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-24T08:37:32.027+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-24T08:37:32.028+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hai, I am newly created task 2 and I will run after the first task!']
[2024-05-24T08:37:32.032+0000] {subprocess.py:86} INFO - Output:
[2024-05-24T08:37:32.033+0000] {subprocess.py:93} INFO - hai, I am newly created task 2 and I will run after the first task!
[2024-05-24T08:37:32.033+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-24T08:37:32.034+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-24T08:37:32.048+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_task, run_id=scheduled__2024-05-14T00:00:00+00:00, execution_date=20240514T000000, start_date=20240524T083731, end_date=20240524T083732
[2024-05-24T08:37:32.083+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-24T08:37:32.091+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
